<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>17</y>
      <width>18</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>53</y>
      <width>18</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="end_hour">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>53</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="begin_minute">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>7</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="singleStep">
     <number>30</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="begin_hour">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>7</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>23</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>时</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="end_minute">
    <property name="geometry">
     <rect>
      <x>158</x>
      <y>53</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>59</number>
    </property>
    <property name="singleStep">
     <number>30</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>分</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ddl战士指数</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>夜猫子指数</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSlider" name="night_changer">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSlider" name="ddl_changer">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#0055ff;&quot;&gt;每日起止&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#0055ff;&quot;&gt;设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>237</y>
     <width>214</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
